src/templates/components/container.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default" ng-controller="formioContainerComponent">
+  <div class="panel-heading" ng-init="component.hideLabel = true">
+    <h3 class="panel-title">{{ component.label}}</h3>
+  </div>
+  <div class="panel-body">
+    <div ng-repeat="containerData in data[component.key] track by $index">
+      <div ng-repeat="component in component.components">
```

Comment:
Thanks Travis for your feedback. I will look into you suggestions and get back to you.